docs/concepts/manifest-builds.md

```diff
@@ -55,6 +55,7 @@ this:
 [build.myproject]
 command = '''
   cargo build --release
+  mkdir -p $out/bin
```

Comment:
nitpick(non-blocking): Coming from the Nix world, I'd expect this line to be one of the first, if not the first. I'm fine with either, just thought I'd mention it.
```suggestion
  mkdir -p $out/bin
  cargo build --release
```

Reply:
I located them just before the command that uses the directory (e.g. `cp`).

Not opposed to moving them, but I wonder if it makes this excerpt less familiar from the previous references if we have to remove the intermediate `cargo build` command. What do you think?

> This is why you see the following line in the build command for the `myproject`
> example above
> 
> ```sh
> mkdir -p $out/bin
> cp target/release/myproject $out/bin/myproject
> ```